.github/workflows/publish.yml

```diff
@@ -1,7 +1,6 @@
 name: Test and Release Go CLI
 
-on:
-  push:
+on: [push]
```

Comment:
This is just a way to write the same things with a different syntax, and I prefer this one.